.github/workflows/markdown-link-check.yml

```diff
@@ -0,0 +1,25 @@
+name: Markdown link check
+
+on:
+  pull_request:
```

Comment:
maybe once a week is fine?

Reply:
As of now it only runs for PRs having markdown changes and takes ~40 secs to complete. But no strong opinion, once a week works fine too if we all agree :)

Reply:
if it's ~40 sec maybe we can just run it for every PR that touches markdown?